pkgs/tools/package-management/nix-index/wrapper.nix

```diff
@@ -0,0 +1,19 @@
+{ lib, symlinkJoin, nix-index-unwrapped, makeWrapper, nix }:
+
+if nix == null then nix-index-unwrapped else
```

Comment:
Stop adding new uses for some package is null! This is only ever the case when someone overrides random package or inputs to null which is totally unsupported for obvious reasons. If you want nix-index unwrapped then just use nix-undex unwrapped instead.

Reply:
Fair